File: app/src/main/java/jp/co/terraresta/androidlesson/data/model/media/ImageUploadData.java
package jp.co.terraresta.androidlesson.data.model.media;

import com.google.gson.annotations.Expose;

import jp.co.terraresta.androidlesson.data.model.common.BaseResultData;

/**
 * Created by ooyama on 2017/05/29.
 */

public class ImageUploadData extends BaseResultData {

    @Expose
    public int imageId;

    public int getImageId() {
        return imageId;
    }
}

File: app/src/main/java/jp/co/terraresta/androidlesson/data/model/media/ImageUploadData.kt
package jp.co.terraresta.androidlesson.data.model.media

import com.google.gson.annotations.Expose

import jp.co.terraresta.androidlesson.data.model.common.BaseResultData

/**
 * Created by ooyama on 2017/05/29.
 */

class ImageUploadData : BaseResultData() {

    @Expose
    var imageId: Int = 0
}
